src/components/SuggestedActionHamburgerMenu/SuggestedActionHamburgerMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import Menu, { MenuProps } from '../Menu';
+
+export interface SuggestedActionHamburgerMenuProps extends MenuProps {
+  open: boolean;
+  anchorEl: HTMLElement | null;
+  onClose: (args?: any) => void;
+}
+
+const SuggestedActionHamburgerMenu = ({
+  open,
+  anchorEl,
+  onClose,
+  children,
+  ...props
+}: SuggestedActionHamburgerMenuProps) => (
+  <Menu
+    open={open}
+    anchorEl={anchorEl}
+    onClose={onClose}
+    anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+    transformOrigin={{ vertical: 'top', horizontal: 'right' }}
+    sx={{ '& .MuiPaper-root': { width: '25%', minWidth: '270px' } }}
```

Comment:
since we probably won't modify styles at the root level it should be fine, but ideally we'd wrap this with the `styled` utility so that a consumer component can pass `sx` if they want without completely overriding the styles defined here

Reply:
nah, just specific to `sx`. it's probably fine to allow fully overriding the origin props--though we usually prob won't need to